static/js/place/place.ts

```diff
@@ -185,19 +185,31 @@ function renderPage(dcid: string) {
     );
   });
 
-  Promise.all([chartConfigDataPromise, parentPlacesPromise]).then(
-    (resolvedValues) => {
+  Promise.all([
+    chartConfigDataPromise,
+    parentPlacesPromise,
+    childPlacesPromise,
+    similarPlacesPromise,
+    nearbyPlacesPromise,
+  ]).then(
+    ([
+      chartConfigData,
+      parentPlaces,
+      childPlaces,
+      similarPlaces,
+      nearbyPlaces,
+    ]) => {
       ReactDOM.render(
         React.createElement(MainPane, {
           dcid,
           placeType,
           topic,
-          chartConfig: resolvedValues[0].config,
-          chartData: resolvedValues[0].data,
-          parentPlaces: resolvedValues[1],
-          childPlacesPromise,
-          similarPlacesPromise,
-          nearbyPlacesPromise,
+          chartConfig: chartConfigData.config,
+          chartData: chartConfigData.data,
+          parentPlaces: parentPlaces,
+          childPlaces: childPlaces,
+          similarPlaces,
```

Comment:
should these get explicit keys too?

Reply:
done

Reply:
seems the shorthand key is preferred, so change all fields to use shorthand key